packages/client-core/src/auth.ts

```diff
@@ -29,7 +36,9 @@ export const getAuthToken = async (
   }
 
   if (auth.scheme === 'bearer') {
-    return `Bearer ${token}`;
+    const format =
+      auth.bearerFormat !== undefined ? auth.bearerFormat : 'Bearer';
```

Comment:
What do you think about checking for any truthy value instead of only undefined?

```suggestion
      auth.bearerFormat || 'Bearer';
```